lib/cli/src/Cardano/CLI.hs

```diff
@@ -1230,13 +1234,26 @@ withLogging configFile minSeverity action = bracket before after (action . snd)
                             Unicode Terminal Helpers
 -------------------------------------------------------------------------------}
 
--- | Override the system output encoding setting. This is needed because the CLI
--- prints UTF-8 characters regardless of the @LANG@ environment variable.
+-- | Force the locale text encoding to UTF-8. This is needed because the CLI
+-- prints UTF-8 characters regardless of the @LANG@ environment variable or any
+-- other settings.
+--
+-- On Windows the current console code page is changed to UTF-8.
 setUtf8Encoding :: IO ()
+#if WINDOWS
 setUtf8Encoding = do
-    setLocaleEncoding utf8
-    hSetEncoding stdout utf8
-    hSetEncoding stderr utf8
+    let utf8CP = 65001 -- code page number for UTF-8
```

Comment:
The comment can probably be made redundant if we rename the identifier to something like:

```utf8CodePage```.

Reply:
Sure